.github/workflows/update.js

```diff
@@ -84,10 +94,22 @@ async function updatePackageJson(version) {
 
 async function commitAndPushTag(version) {
   const tag = `v${version}`;
+  const message = getMessage(version);
   await git("add", "README.md", "package.json", "package-lock.json");
-  await git("commit", "-m", `"MAINT: upgrade to ${PACKAGE_NAME} ${version}"`);
-  await git("tag", tag);
-  await git("push", "origin", tag);
+  await git("commit", "--message", `"${message}"`);
+  await git("tag", "--message", `"${message}"`, "--annotate", tag);
+}
+
+async function mergePullRequest(versions) {
+  await git(
+    "push",
+    "--follow-tags",
+    "--set-upstream",
+    "origin",
+    getBranchName(versions.at(-1))
+  );
+  await exec(`gh pr create --fill --title "${getMessage(versions.at(-1))}"`);
+  await exec("gh pr merge --merge --delete-branch");
```

Comment:
Depending on the branch protection rules, whether PR approvals or status checks is required, this command may need to be modified accordingly.

```suggestion
  await exec("gh pr merge --auto --merge --delete-branch");
```

Also need to ensure that PR merge commits are allowed in the repo.